core/lomas_core/error_handler.py

```diff
@@ -121,5 +133,14 @@ async def internal_server_exception_handler(
         LOG.info(f"InternalServerException raised: {exc.error_message}")
         return JSONResponse(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            content={"InternalServerException": INTERNAL_SERVER_ERROR},
+            content=jsonable_encoder(InternalServerExceptionModel()),
         )
+
+
+# Server error responses for DP queries
+SERVER_QUERY_ERROR_RESPONSES: dict[int | str, dict[str, Any]] = {
```

Comment:
What is the goal of this and ...ExceptionModel exceptions versus what was there before ?

Reply:
This allows to list the possible responses that an endpoint can return:
```
@router.post(
    "/dummy_diffprivlib_query",
    dependencies=[Depends(server_live)],
    response_model=QueryResponse,
    responses=SERVER_QUERY_ERROR_RESPONSES,
    tags=["USER_DUMMY"],
)
```

The format of the dict in responses requires pydantic models. Using these also has the benefit that we can show the output model in the redoc documentation, not only for the standard response, but also for exceptions.